### PR TITLE
fetch/update/delete currently authorized user

### PR DESCRIPTION
**Pull request by KutsenkoA**
- current user management (fetch, update password, delete)
- e2e tests

task: [https://jexia.myjetbrains.com/youtrack/issue/JEXIA-1584](url)

### CODE REVIEW

File: extra_docs/ums/ums_draft.md [hunk 1]
```diff
@@ -92,21 +92,18 @@ ums.activate({
 });
 ```
 
-### Get user by id
+### Fetch user by auth alias (or email)
 ```javascript
-ums.getUserById(userId); // -> return user or throw an error
```
Comment (marlosin) [inline extra_docs/ums/ums_draft.md]: Elon Mask

File: src/api/ums/umsModule.spec.ts [hunk 2]
```diff
@@ -147,4 +147,91 @@ describe('UMS Module', () => {
     });
   });
 
+  describe('user management', () => {
+
+    describe('get current user', () => {
+      it('should get token from token manager by provided alias', async () => {
+      const { subject, tokenManagerMock, systemDefer, init } = createSubject({
```
Comment (marlosin) [inline src/api/ums/umsModule.spec.ts]: maybe you forgot to remove the unnecessary parameter: `createSubject()`

File: src/api/ums/umsModule.spec.ts [hunk 3]
```diff
@@ -147,4 +147,91 @@ describe('UMS Module', () => {
...
+      });
+      jest.spyOn(tokenManagerMock, 'token');
+      systemDefer.resolve();
+      await init();
+      await subject.getUser(testUser.auth);
+      expect(tokenManagerMock.token).toHaveBeenCalledWith(testUser.auth);
+    });
+
+      it('should call correct API to get current user', async () => {
```
Comment (marlosin) [inline src/api/ums/umsModule.spec.ts]: messy indentation block

File: extra_docs/ums/ums_draft.md (hunk 1 above)
Comment (KutsenkoA) [reply extra_docs/ums/ums_draft.md]: :-D

**Pull request merged**